stmocli/cli.py

```diff
@@ -40,6 +42,12 @@ def track(conf, query_id, file_name, redash_api_key):
             urljoin(redash.API_BASE_URL, url_path)
         )
 
+        if not file_name:
```

Comment:
Can we make this more explicit by using a [prompt](http://click.pocoo.org/5/prompts/) with the default set to your generated filename?

Reply:
Will do.